types/office-js/index.d.ts

```diff
@@ -20447,7 +20447,7 @@ declare namespace Office {
          * The `commandId` value must match the task pane ID specified in the manifest of your add-in. The markup depends on the type of manifest your
          * add-in uses.
          *
-         * - **XML manifest**: The `id` attribute of the {@link https://learn.microsoft.com/javascript/api/manifest/control | Control} element representing the task pane.
+         * - **Add-in only manifest**: The `id` attribute of the {@link https://learn.microsoft.com/javascript/api/manifest/control | Control} element representing the task pane.
          *
          * - **Unified manifest for Microsoft 365 (preview)**: The "id" property of the task pane command in the "controls" array.
```

Comment:
Checking if this should still be indicating it's in preview...

```suggestion
         * - **Unified manifest for Microsoft 365**: The "id" property of the task pane command in the "controls" array.
```